src/core/globals.d.ts

```diff
@@ -5,4 +5,10 @@ declare global {
     // Global property set by test some harnesses to signal karma testing environment.
     __karma__?: boolean;
   }
+
+  interface Document {
```

Comment:
Let's move this to `#core/amp-globals.d.ts` instead

Reply:
Should we also move `BaseElement` and `AMP` definitions here? (future PR)

Reply:
BaseElement probably AMP definitely. We've done that in most places so far, splitting globals from AMP-specific globals